doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -83,41 +83,76 @@ constraints they must verify (for :g:`pidentity` there are no constraints).
 
 Inductive types can also be declared universes polymorphic on
```

Comment:
Unrelated to the present changes, but since I noticed it:

```suggestion
Inductive types can also be declared universe polymorphic on
```

Reply:
I have this in https://github.com/coq/coq/pull/15981/files